libsrc/vic20/kernal.s

```diff
@@ -47,9 +47,9 @@
 ;-----------------------------------------------------------------------------
 ; All functions are available in the kernal jump table
 
-CINT            = $FF81
-IOINIT          = $FF84
-RAMTAS          = $FF87
+CINT            = $E518         ; No entries are in the kernal jump table for these functions.
+IOINIT          = $FDF9         ; The entries point directly to the function.
+RAMTAS          = $FD8D         ;
```

Comment:
I think that the second comment should be moved down to the third line.  Otherwise, someone might think that the third semicolon is a stray (it might be removed).  Also, I think that the word "function" in that line should be plural ("functions").

Other than those two little things, this PR is a "no-brainer"; it can be merged, as soon as those things are changed.